src/seedu/addressbook/data/person/Block.java

```diff
@@ -0,0 +1,29 @@
+package seedu.addressbook.data.person;
+
+/**
+ * Represents a Person's block in the address book.
+ */
+public class Block {
+
+    private String block;
+
+    public Block(String block) {
+        this.block = block;
+    }
+
+    public String getBlock() {
```

Comment:
Is it necessary to have this getter function in this case if `toString()` also fulfils the same purpose? Same for `Unit`, `Street` and `Postal Code`...